### PR TITLE
New package: idevicerestore libirecovery

### PR DESCRIPTION
**Pull request by CRKatri**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by kawaiiamber** [comment]
Short descriptions shouldn't start with articles and shouldn't end in periods. Try changing to `Allows communication with iBoot/iBSS of iOS devices via USB` and `Command-line application to restore firmware files to iOS devices`

File: srcpkgs/idevicerestore/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'idevicerestore'
+pkgname=idevicerestore
+version=1.0.0
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake libtool pkg-config"
+makedepends="libimobiledevice-devel libplist-devel libressl-devel
+ libzip-devel readline-devel libcurl-devel libirecovery-devel"
+depends="usbmuxd"
+short_desc="A command-line application to restore firmware files to iOS devices."
```
Comment (kawaiiamber) [inline srcpkgs/idevicerestore/template:10]: `short_desc=Command-line application to restore firmware files to iOS devices`

File: srcpkgs/libirecovery/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'libirecovery'
+pkgname=libirecovery
+version=1.0.0
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake libtool pkgconf"
+makedepends="libusb-devel readline-devel ncurses-devel eudev-libudev-devel"
+short_desc="The libirecovery library allows communication with iBoot/iBSS of iOS devices via USB."
```
Comment (kawaiiamber) [inline srcpkgs/libirecovery/template:8]: `short_desc=Allows communication with iBoot/iBSS of iOS devices via USB`

File: srcpkgs/idevicerestore/template (hunk 1 above)
Comment (kawaiiamber) [inline srcpkgs/idevicerestore/template:10]: `short_desc=Command-line application to restore firmware files to iOS devices`

**Review by kawaiiamber — CHANGES_REQUESTED** [review]
To pass the link check

**Comment by ericonr** [comment]
Ideally you wouldn't add the packages as orphaned...

**Comment by Jackojc** [comment]
Any updates on getting these packages merged?

**Comment by ahesford** [comment]
The first step is not creating new packages that are orphaned from the outset.

**Pull request closed without merging**